include/battle_script_commands.h

```diff
@@ -45,6 +46,7 @@ void StealTargetItem(u8 battlerStealer, u8 battlerItem);
 u8 GetCatchingBattler(void);
 u32 GetHighestStatId(u32 battlerId);
 bool32 ProteanTryChangeType(u32 battler, u32 ability, u32 move, u32 moveType);
+bool32 DoesSkyBattleCancelMove(u32 gCurrentMove);
```

Comment:
bool32 DoesSkyBattleCancelMove(u32 move);

gCurrentMove is an already existing global variable.